react/.npmrc

```diff
@@ -0,0 +1,2 @@
+engine-strict=true
+ignore-scripts=true
```

Comment:
When including this, we should make sure that no packages require to run scripts in order to work. As of now, we don't have any packages requiring this, but we should make sure we don't install any. Maybe we could make use of https://github.com/dominykas/allow-scripts? The question is then how we can whitelist packages? npm does not support anything like that yet.